app/controllers/events_controller.rb

```diff
@@ -200,6 +201,24 @@ def destroy
     end
   end
 
+  def copy_to_conference
+    old_event = Event.find(params[:id])
+    new_event = old_event.dup
+    authorize! :create, new_event
+
+    new_conference = Conference.find(params[:event][:conference_id])
+    new_event.conference = new_conference
+    new_event.event_attachments << old_event.event_attachments
+    new_event.event_people << old_event.event_people
```

Comment:
`ActiveRecord::Base#dup` creates a shallow copy, clearing associations. I'd say it will just link all `old_event.event_*` to the new event. Thus it is **not a deep clone** and would mess with conferences history/state by *"stealing"* associated objects which hold a foreign key to a conference.